app/hotels/src/GraphQLSanitizers.js

```diff
@@ -6,7 +6,7 @@ export const sanitizeDate = (input: ?Date): ?string => {
   return input && DateFormatter(input).formatForMachine();
 };
 
-export const sanitizeHotelFacilities = (hotelFacilities: string[]) => {
+export const sanitizeHotelAmenities = (hotelFacilities: string[]) => {
```

Comment:
Sorry to be picky, but the variable name should probably also be renamed to erase `Facilities` entirely.

Reply:
Picky is good, I will fix it